test/integration/pipeline/tx_pipeline_integration_test.cpp

```diff
@@ -72,11 +83,48 @@ class TxPipelineIntegrationTest : public TxPipelineIntegrationTestFixture {
     std::remove("admin@test.priv");
     std::remove("test@test.pub");
     std::remove("test@test.priv");
+    std::remove("sample@test.pub");
+    std::remove("sample@test.priv");
+    std::remove("example@test.pub");
+    std::remove("example@test.priv");
   }
+
+  iroha::model::Transaction generateCreateUsersTransaction() {
```

Comment:
I am not sure that, such generation method is required. Maybe better to use shared_model transaction and backward converter?